addons/web/static/src/search/with_search/with_search.js

```diff
@@ -116,6 +106,20 @@ WithSearch.props = {
         },
         optional: true,
     },
+    viewSwitcherEntries: {
```

Comment:
not sure about adding viewSwitcherEntries and breadcrumbs to the searchmodel.

Reply:
It's to avoid passing props the the layout component, although @Polymorphe57 suggested we use a separate object in the `env` to contain all the view/action-specific information, to avoid overloading the SearchModel. I think this could be a discussion for an ulterior PR